ImplementationGuide/markdown/ISiKundSMART/Schritt2ClientAutorisierung.md

```diff
@@ -46,7 +42,7 @@ Als Einstiegspunkt für einen Standalone Launch muss dem Client die Url des FHIR
 
 ## Hinweise zu Identity Scopes
 
-Um Informationen über die authentifizierten Endbenutzer:in zu erhalten, kann ein Client per OpenID Connect ein Identitätstoken zusammen mit einem Zugangstoken anfragen. Hierzu sind in Kombination die Scopes "openid" und "fhirUser" zu verwenden. EHR müssen die Vorgaben nach [2.0.4 - Scopes for requesting identity data](https://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html#scopes-for-requesting-identity-data) umsetzen. Anstatt dem Claim "fhirUser" kann - aus Gründen der Rückwärtskompatibilität - auch der Claim "profile" verwendet werden.  
+Um Informationen über die authentifizierten Endbenutzer:in zu erhalten, kann ein Client per OpenID Connect ein Identitätstoken zusammen mit einem Zugangstoken anfragen. Hierzu sind in Kombination die Scopes "openid" und "fhirUser" zu verwenden. EHR müssen die Vorgaben nach [3.0.4 - Scopes for requesting identity data](https://hl7.org/fhir/smart-app-launch/STU2.1/scopes-and-launch-context.html#scopes-for-requesting-identity-data) umsetzen. Anstatt dem Claim "fhirUser" kann - aus Gründen der Rückwärtskompatibilität - auch der Claim "profile" verwendet werden.  
```

Comment:
suggestion: die Spec gendert an den meisten Stellen nicht, sodass "Endbenutzer:in" in diesem Absatz nicht konsequent erscheint. Entweder Endbenutzer oder Endbenutzer:innen